assets/js/application.js

```diff
@@ -0,0 +1,75 @@
+if (document.readyState !== "loading") { initTransitionView() }
+document.addEventListener("DOMContentLoaded", initTransitionView);
+
+function initTransitionView() {
+  if (screen.width > 700) {
+    const navbar_links = document.querySelectorAll(".transition-link")
```

Comment:
I believe it's more convenient to use IDs for JavaScript logic rather than classes.

Reply:
or data attributes